docs/rule/no-whitespace-within-word.md

```diff
@@ -0,0 +1,53 @@
+## no-whitespace-within-word
+
+Using in-line whitespace word formatting results in pages that are in violation of the Web Content Accessibility Guidelines (WCAG 2.0). This approach reflects a choice to ignore [WCAG's officially-documented techniques](https://www.w3.org/TR/WCAG20-TECHS/intro.html) that ensure web accessibility criteria are met, and, unsurprisingly, produces results that are explicitly mentioned in [WCAG's list of common sources of web accessibility failures](https://www.w3.org/TR/WCAG20-TECHS/failures.html).
```

Comment:
> Using in-line whitespace word formatting results in pages that are in violation of the Web Content Accessibility Guidelines (WCAG 2.0).

Can we make the case better than "XYZ spec says so"? In theory, the spec is well reasoned, and we can inline some of that basic reasoning here while still referencing the spec. 

> This approach reflects a choice to ignore [WCAG's officially-documented techniques](https://www.w3.org/TR/WCAG20-TECHS/intro.html) that ensure web accessibility criteria are met, and, unsurprisingly, produces results that are explicitly mentioned in [WCAG's list of common sources of web accessibility failures](https://www.w3.org/TR/WCAG20-TECHS/failures.html)

Lets avoid:

* assumption of malice or willful ignorance
* condescending tone